Makefile

```diff
@@ -49,6 +49,7 @@ ct-coverage-lfvm: TOSCA_GO_COVERAGE_EVM=lfvm
 ct-coverage-lfvm: coverage-go
 
 ct-coverage-geth: TOSCA_GO_COVERAGE_EVM=geth
+ct-coverage-geth: TOSCA_GO_COVERAGE_DEPENDENCY_PACKAGES=github.com/ethereum/go-ethereum/core/vm/...
```

Comment:
Shall we add more packages here? 
I think we use vm, core, common, params, and types. 

I am not sure if adding github.com/ethereum/go-ethereum would be a good idea, as we may no use the most of it.

Reply:
As discussed, other packages are not relevant. important stuff is in vm.

Reply:
I think that all the relevant parts for us, are just in core/vm, and this should be sufficient. 
As discussed offline, the gas expantion cost and instructions are both in vm package, so this is fine.